src/app/shared/user/user.service.ts

```diff
@@ -38,6 +40,17 @@ export class UserService {
     });
   }
 
+  addUserToWorkflows(user: User): void {
+    this.alertService.start('Adding user to existing workflows on Dockstore');
+    this.usersService.addUserToDockstoreWorkflows(user.id).subscribe(
```

Comment:
Hmm...surprised this worked, thought swagger was disabled for new endpoints.  In any case, use the openapi service and model, cast the models for compatibility.

Reply:
I couldn't get patch to work without having both swagger and OpenAPI3 annotations, but ya I'll change to openapi.

Reply:
I cannot add both the usersService swagger and usersService openAPI in the same file. Other endpoints being called are not implemented for openapi, so I am going to keep this as swagger.

Reply:
Not sure how I didn't notice the swagger PATCH 

`import { UsersService as OpenAPIUsersService } from '../../../shared/openapi';`

We use OpenAPI for a lot of new endpoints.